testing/radicale/APKBUILD

```diff
@@ -1,32 +1,22 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=radicale
+_pkgname=Radicale
 pkgver=2.1.8
 pkgrel=1
 pkgdesc="A simple CalDAV (calendar) and CardDAV (contact) server"
 url="http://radicale.org"
 arch="noarch"
 license="GPLv3+"
 depends="python3 py3-vobject"
-depends_dev=""
-makedepends="$depends_dev python3"
+makedepends="python3"
 pkgusers="radicale"
 pkggroups="radicale"
 install="$pkgname.pre-install"
-subpackages=""
-source="https://files.pythonhosted.org/packages/source/R/Radicale/Radicale-$pkgver.tar.gz
+options="!check" # no test suite
+source="https://files.pythonhosted.org/packages/source/R/$_pkgname/$_pkgname-$pkgver.tar.gz
```

Comment:
I prefer not use `$_pkgname` at all. It makes copy/paste the source URL to web browser harder.